wrapanapi/systems/openstack.py

```diff
@@ -634,9 +634,23 @@ def tenant_api(self):
 
     @property
     def capi(self):
-        if not self._capi:
-            self._capi = cinderclient.Client(session=self.session, service_type="volume")
-        return self._capi
+        if self._capi:
+            return self._capi
+
+        # Detect the higest possible hopefully working cinder version
+        service_types = 'volumev3 volumev2 volume'.split()
```

Comment:
Please define this as a list instead of calling split() on the string.

Reply:
Well I like string for its readability, but I don't insist on it.